evm/vm/forks/byzantium/__init__.py

```diff
@@ -2,7 +2,8 @@
     merge,
 )
 
-from evm import constants
+from evm.constants import MAX_UNCLE_DEPTH
+from .constants import EIP649_BLOCK_REWARD
```

Comment:
Can you go take a look at [what the style guide has to say about import ordering](https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#import-statements).

These `.constants` imports should be further down below all of the `evm.` level imports since they are *local*.